sdk/include/opentelemetry/sdk/metrics/export/periodic_exporting_metric_reader.h

```diff
@@ -0,0 +1,69 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include "opentelemetry/sdk/metrics/metric_reader.h"
+#  include "opentelemetry/version.h"
+
+#  include <atomic>
+#  include <chrono>
+#  include <condition_variable>
+#  include <thread>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace metrics
+{
+
+class MetricExporter;
+/**
+ * Struct to hold batch PeriodicExortingMetricReader options.
+ */
+
+struct PeriodicExportingMetricReaderOptions
+{
+
+  /* The time interval between two consecutive exports. */
+  std::chrono::milliseconds schedule_delay_millis = std::chrono::milliseconds(60000);
```

Comment:
The name `schedule_delay_millis` might be misleading.

The suggested name is `export_interval_millis` https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#periodic-exporting-metricreader

Reply:
For example, if export 1 runs during [T1, T2] and export 2 runs during [T3, T4], `export_interval_millis = T3 - T1`, while `schedule_delay_millis = T3 - T2`.

Reply:
Thanks, changed the variable name to `export_interval_millis`, and also logic to ensure there is exactly `schedule_delay_millis` ms difference between two exports ( by adjusting the time taken for collect and export operation ).